app/src/main/java/com/sun/homecinema/data/model/Actor.kt

```diff
@@ -0,0 +1,29 @@
+package com.sun.homecinema.data.model
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import com.google.gson.annotations.SerializedName
+import com.sun.homecinema.data.model.Actor.Companion.TABLE_ACTOR
+
+@Entity(tableName = TABLE_ACTOR)
+data class Actor(
+    @SerializedName(KEY_ID)
+    @PrimaryKey
+    val castId: Long,
+    var castMovieId: Long,
+    val name: String,
+    @SerializedName(KEY_AVATAR)
+    val avatar: String
+): GeneraEntity {
+    override fun areItemsTheSame(newItem: GeneraEntity): Boolean =
+        this.castId == (newItem as Actor).castId
```

Comment:
đầu tiên phải check xem nó có phải là Actor không đã chứ 
```suggestion
        newItem is Actor && this.castId == newItem.castId
```